### PR TITLE
parser:add more benchmarks with different workload

### PR DESCRIPTION
**Pull request by zhexuany**
@coocood @shenli PTAL

### CODE REVIEW

**Comment by coocood** [comment]
The name is misleading.
`parseInt` means only parse int, not a complete select statement that contains an int value.

File: parser/bench_test.go [hunk 1]
```diff
@@ -17,34 +17,103 @@ import (
 	"testing"
 )
 
-func BenchmarkSysbenchSelect(b *testing.B) {
+func sysebchSelect(parser *Parser, sql string, num int, b *testing.B) {
```
Comment (coocood) [inline parser/bench_test.go]: This function has nothing to do with `sysbench`.

Comment (zhexuany) [reply parser/bench_test.go]: As you can see, old function name declared as `SysbenchSelect`. I just adopted same name.

Comment (coocood) [reply parser/bench_test.go]: It's an extracted function, it run parsing for N times and doesn't have to be called by BenchmarkSysbenchSelect.

File: parser/bench_test.go [hunk 2]
```diff
@@ -17,34 +17,103 @@ import (
...
+	for i := 0; i < num; i++ {
+		_, err := parser.Parse(sql, "", "")
+		if err != nil {
+			b.Fatal(err)
+		}
+	}
+}
+
+func BenchmarkSysbenchSelect1024(b *testing.B) {
```
Comment (coocood) [inline parser/bench_test.go]: We don't need multiple benchmarks to run a different number of loops.

Comment (zhexuany) [reply parser/bench_test.go]: The purpose is to simulate different workloads. My way to do this is wrong?

Comment (coocood) [reply parser/bench_test.go]: The workloads are not different.

Comment (zhexuany) [reply parser/bench_test.go]: but `testing.B` thinks it is different workload. `testing.B` will run enough time to make sure the accurateness. If we us this, `testing.B` thinks its a operation. In my understanding, it is different workload. If my understanding is wrong, then what is correct way to do it?

Comment (coocood) [reply parser/bench_test.go]: If you want the benchmark run longer to get a more accurate result, you can specify the time to run. like `-benchtime=10s`.

**Pull request closed without merging**